README.rst

```diff
@@ -2,34 +2,20 @@
 
 
 These configuration files facilitate ``pre-commit`` to maintain ``black`` formatting, ``flake8`` compliance and ``isort`` imports sorting.
-The configuration files are downloaded by using the ``pre-commit-conf`` command from the ``centralized-pre-commit-conf`` project.
-The maintainer of the ``centralized-pre-commit-conf`` project only makes available installation using ``pip``.
-Since TSSW uses ``conda`` (or ``mamba``) for their software installations, a conda recipe was created for ``centralized-pre-commit-conf`` and the conda package was made available in the ``lsstts`` conda channel.
+If ``mypy`` is used, a configuration file can be generated for that as well.
+The configuration files are generated by using the ``generate_pre_commit_conf`` command.
+Since TSSW uses ``conda`` (or ``mamba``) for their software installations, a conda recipe was created for this project and the conda package was made available in the ``lsstts`` conda channel.
 To enable this:
 
 * Install pre-commit, for instance ``conda install -y pre-commit``.
-* Install centralized-pre-commit-conf, for instance ``conda install -y -c lsstts centralized-pre-commit-conf``.
-* Then either:
-
-    * Download the ``config.yaml`` file of your choice from the ``develop`` directory in the ``develop`` branch on ``https://github.com/lsst-ts/ts_pre_commit_conf/`` and place it in ``~/.config/pre-commit-conf`` on UNIX systems (Linux and macOS).
-    * Run ``pre-commit-conf -f`` once in every project to download the configuration files for pre-commit and add them to ``.gitignore``.
-
-  OR
-
-    * Download the configuration files using ``pre-commit-conf -f -r "https://github.com/lsst-ts/ts_pre_commit_conf/" -b "develop" -u -p "develop" -c .flake8 .isort.cfg .mypy.ini .pre-commit-config.yaml``.
-
-Note that the ``develop`` directory and the ``-p "develop"`` command line argument are referring to the same path in the ``ts_pre_commit_conf`` repo.
-Other directories may be available as well.
-The ``-b "develop"`` command line argument refers to the branch in the ``ts_pre_commit_conf`` repo.
-For developers actively helping to maintain the configuration files, other branches can be used to pull the configuration files for testing purposes.
-It is not recommended to use a different branch for other projects!
-
-For more information about the use of the ``pre-commit-conf`` command, see the `centralized-pre-commit-conf <https://github.com/Pierre-Sassoulas/centralized-pre-commit-conf/>`_ homepage.
-The ``appropriate search paths`` link on that page is wrong and should point `here <https://confuse.readthedocs.io/en/latest/usage.html#search-paths>`_.
+* Install ``ts_pre_commit_conf``, for instance ``conda install -y -c lsstts ts-pre-commit-conf``.
+* Then execute the ``generate_pre_commit_conf`` command.
+  Note that this command accepts the ``-h`` or ``--help`` argument that will display usage information.
```

Comment:
Please supplement this to tell a user how to update these files when the ts_pre_commit_conf package is updated to require different versions of the various pre-commit hooks.